corehq/apps/data_dictionary/views.py

```diff
@@ -283,6 +275,18 @@ def _get_case_data(case_type):
     return JsonResponse(case_type_data)
 
 
+def _get_pagination_links(current_url, total_records, skip, limit):
+    links = {"self": update_url_query_params(current_url, {"skip": skip, "limit": limit})}
```

Comment:
Isn't `self` just `current_url`?

Reply:
Good question. Since skip and limit are optional parameters, this ensures they are added to the url when they are not provided.